src/Extractor/BaseExtractor.php

```diff
@@ -54,7 +56,7 @@ public function __construct(array $parameters, array $state, LoggerInterface $lo
         }
         $this->dbParameters = $parameters['db'];
 
-        $proxy = new DbRetryProxy($this->logger, self::DEFAULT_MAX_TRIES, [PDOException::class]);
+        $proxy = new DbRetryProxy($this->logger, self::CONNECT_MAX_RETRIES, [PDOException::class]);
```

Comment:
Pri vytvarani spojenia sa nepouziva `retries` hodnota z configu ale defaultna hodnota `5` ... funkcionalitu som nechal nezmenenú, ... no premenoval som to, nech je to jasnejsie.